sycl/source/detail/graph_impl.hpp

```diff
@@ -243,6 +243,19 @@ class node_impl : public std::enable_shared_from_this<node_impl> {
       return createCGCopy<sycl::detail::CGFillUSM>();
     case sycl::detail::CGType::PrefetchUSM:
       return createCGCopy<sycl::detail::CGPrefetchUSM>();
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
```

Comment:
Is there a reason why getCGCopy() throws an exception here?

In command.cpp line 3046 there is an implementation for these new command group types when using Graphs. If that implementation is working, then I wouldn't expect an exception to be thrown only when a CG copy happens.